packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2258,34 +2291,40 @@ class InputDecoration {
 
   @override
   int get hashCode {
+    // split into multiple hashValues calls
```

Comment:
This is good, however please make it into more of a sentence. Like:  `Split into multiple hashValues calls because the hashValues function is limited to 20 parameters.`